doxygen/dox/ViewTools.dox

```diff
@@ -997,7 +997,7 @@ In other words, it is an array of four elements, in which each element is a 3 by
 
 This dataset is much more complex. Also note that subsetting cannot be done on Array datatypes.
 
-See this <a href="https://portal.hdfgroup.org/display/knowledge/H5T_ARRAY+Datatype">FAQ</a> for more information on the Array datatype.
+See this <a href="https://docs.hdfgroup.org/hdf5/develop/_l_b_datatypes.html">section</a> for more information on the Array datatype.
```

Comment:
Should this be a versioned link for the 1.14 branch or is develop fine to point to?

Reply:
Yes.  Binh-Minh said to watch for that, but I forgot.  I'll fix it.  I think it's 114 for both hdf5_1_14 and hdf5_1_14_4, but I'll check.